apps/andi/lib/andi/dataset_cache.ex

```diff
@@ -33,7 +53,8 @@ defmodule Andi.DatasetCache do
     # In this case Brook is already single threaded so it should be ok.
     pid = :ets.new(__MODULE__, [:set, :public, :named_table])
 
-    DatasetRetrieval.get_all!() |> put_datasets()
+    Brook.get_all_values!(instance_name(), :dataset) |> put()
```

Comment:
Why did you go away from using a service here?

Reply:
Keep the same level of abstraction between the two lines. Doesn't make sense to directly use brook in one line and then use a wrapper service in the next line.

Reply:
:+1: I feel like that suggests both calls to Brook should be in the service layer